databases/week1/class/homework.sql

```diff
@@ -0,0 +1,83 @@
+-- Find out how many tasks are in the task table
+SELECT 
+  COUNT(title) 
+FROM 
+  task;
+  
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT 
+  COUNT(title) 
+FROM 
+  task 
+WHERE 
+  due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT 
+  * 
+FROM 
+  task 
+  JOIN status ON task.status_id = status.id 
+WHERE 
+  status.name LIKE "%done%";
+
+-- Find all the tasks that are not marked as done
+SELECT 
+  * 
+FROM 
+  task 
+  JOIN status ON task.status_id = status.id 
+WHERE 
+  NOT status.name LIKE "%done%";
+
+-- Get all the tasks, sorted with the most recently created first
+SELECT 
+  * 
+FROM 
+  task 
+ORDER BY 
+  created DESC;
+
+-- Get the single most recently created task
+SELECT 
+  MAX(created) 
+FROM 
+  task;
+
```

Comment:
SELECT
*
FROM
task
ORDER BY created DESC
LIMIT 1;

_Pointer_
Objective is to find the task that was the most recently created.